tests/test_expertise_service.py

```diff
@@ -528,6 +528,81 @@ def test_fetch_high_load_results(self, openreview_context, celery_session_app, c
                 assert score >= 0 and score <= 1
             assert not os.path.isdir(f"./tests/jobs/{id}")
 
+    def test_request_group_group(self, openreview_context, celery_session_app, celery_session_worker):
+        # Submit a working job and return the job ID
+        MAX_TIMEOUT = 600 # Timeout after 10 minutes
+        test_client = openreview_context['test_client']
+        # Make a request
+        response = test_client.post(
+            '/expertise',
+            data = json.dumps({
+                    'name': 'test_run',
+                    'match_group': ["ABC.cc"],
+                    'submission_group': 'ABC.cc',
```

Comment:
I'm not sure about `submission_group` name, we are trying to support the computation of scores of groups vs group. We should hide to the user the implementation about using one of the group as submissions. 

Why the `match_group` is an array? What happens if there is more than one element?

Can we use match_group array to represent both groups? first element is the match_group and the second element is the other group represented as submissions.

what do you think?

Reply:
Sure, I can rename it from `submission_group` to something more fitting - we can discuss this.

`match_group` has always been able to accept a list of group IDs - in the documentation for the config file:
`"match_group": ["ICLR.cc/2020/Conference/Reviewers", "ICLR.cc/2020/Conference/Area_Chairs"],`
which just retrieves all the profiles from all group IDs to fetch their publications. I could change this to use your suggestion but I would have to move the functionality of grabbing from multiple groups somewhere else.